packages/fx-core/src/component/utils.ts

```diff
@@ -485,9 +485,8 @@ export function getHostingComponent(
   return undefined;
 }
 
-// TODO:implement after V3 project setting update
-export function isHostedByAzure(context: ContextV3): boolean {
-  return true;
+export function isDebug(context: ContextV3): boolean {
```

Comment:
`isDebug` sounds like checking development mode or production mode. Maybe `isLocalEnv` would be better.

Reply:
Updated it. Thanks.